addons/mail/static/src/components/attachment_card/attachment_card.xml

```diff
@@ -22,11 +22,19 @@
                                 <t t-esc="attachmentCard.attachment.displayName"/>
                             </div>
                         </t>
-                        <t t-if="attachmentCard.attachment.extension">
-                            <div class="o_AttachmentCard_extension text-uppercase">
-                                <t t-esc="attachmentCard.attachment.extension"/>
-                            </div>
-                        </t>
+                        <div class="d-flex justify-center">
+                            <t t-if="attachmentCard.attachment.extension">
+                                <div class="o_AttachmentCard_extension text-uppercase">
+                                    <t t-esc="attachmentCard.attachment.extension"/>
+                                </div>
+                            </t>
+                            <div t-if="attachmentCard.attachment.extension and attachmentCard.attachment.size" class="o_AttachmentCard_extension mr-1 ml-1"> - </div>
+                            <t t-if="attachmentCard.attachment.displaySize">
+                                <div class="o_AttachmentCard_extension text-uppercase">
```

Comment:
It's not "extension" part but it displays size, so should name the part appropriately